src/components/Modal.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import general from '../style/general.module.css';
+import modal from '../style/modal.module.css';
+import { Bar } from 'react-chartjs-2';
+import { comparePokemon } from '../redux/actions/compareAction';
+
+const Modal = ({select, compare, comparePokemon}) => {
```

Comment:
this component has a lot of responsibilities, (draw the graph, show the information get the information from the store, trigger the compare) divide the responsibility in multiple components, like one for the graph and other for the painting the general info and other to trigger the comparison